### ISSUE TITLE
Code Contribution: 【Easy】【Operator Development】celu

### ISSUE THREAD

**Issue by 0x45f** [FlagOpen/FlagGems#887]
## Description 任务介绍
Develop forward function for celu operator.
开发celu算子的前向功能。
## Requirements 任务要求
__Interface 接口__
`celu(Tensor self, Scalar alpha=1.0) -> Tensor`
__Function reference 功能参考__
https://docs.pytorch.org/docs/stable/generated/torch.nn.CELU.html
__Implementation reference 实现参考__
https://github.com/FlagOpen/FlagGems/blob/master/src/flag_gems/ops/relu.py

The operator should support all optional arguments defined in the interface. 
算子应支持接口中定义的所有参数选项。
## DDL 提交时间
Please submit a Pull Request within 2 weeks after accepting the assignment.
请于接取任务后两周内提交PR。
Please provide both accuracy test and performance test code.
请同时提供实现正确性测试与性能测试代码。

**Comment by yuxingcyx** [comment]
yuxingcyx 认领

**Comment by xingyanwen70-jpg** [comment]
@yuxingcyx Thank you for your Contibution! Please give me your E-mail Address, I'll give you an E-certificate od honor.
